typescript/cli/src/validator/utils.ts

```diff
@@ -38,21 +42,21 @@ export const getValidatorStorageLocations = async (
 export const getLatestValidatorCheckpointIndexAndUrl = async (
   s3StorageLocation: string,
```

Comment:
would be good to rename this while we're here too

s3StorageLocation -> storageLocation